btf_interp/src/lib.rs

```diff
@@ -1,18 +1,24 @@
 //! Provide interpreter implementation for BF program.
 use btf_types::BrainFuckProgram;
-use std::num::NonZeroUsize;
+use std::{num::NonZeroUsize, cell::Cell};
 
+/// Provide trait for cell in Virtual Machine.
 pub trait CellKind {
-    fn wrapped_add(&mut self, v: &Self) -> Self;
-    fn wrapped_sub(&mut self, v: &Self) -> Self;
+    /// Wrapper to increase value in the cell.
+    fn wrapping_increment(&mut self, v: &Self) -> Self;
+    /// Wrapper to decrease value in the cell.
+    fn wrapping_decrement(&mut self, v: &Self) -> Self;
```

Comment:
I was more expecting:

```suggestion
    /// Increment the cell by 1, wrapping back to zero from max.
    fn wrapping_increment(&mut self);
    /// Decrement the cell by 1, wrapping from zero to max.
    fn wrapping_decrement(&mut self);
```